### ISSUE TITLE
Not actually a fork.

### ISSUE THREAD

**Issue by andybak** [Lumak/Unity5Effects#1]
You seem to have created a completely new repo. It's a shame because as well as losing the original history; the lovely visualization on the Github "fork" page won't work now.

**Comment by Lumak** [comment]
Yes, that's true. There's two unity repos that I merely restored with what I had on my hard drive but will be deleting them and actually properly forking them.
